bin/ch/WScriptJsrt.cpp

```diff
@@ -325,11 +334,46 @@ JsErrorCode WScriptJsrt::InitializeModuleInfo(JsValueRef specifier, JsModuleReco
     return JsNoError;
 }
 
-JsErrorCode WScriptJsrt::LoadModuleFromString(LPCSTR fileName, LPCSTR fileContent)
+bool WScriptJsrt::GetFullPath(LPCSTR fileName, __out char* const fullPathNarrow)
+{
+    size_t len = 0;
+    if (fileName)
+    {
+        if (_fullpath(fullPathNarrow, fileName, _MAX_PATH) == nullptr)
+        {
+            return false;
+        }
+        // canonicalize that path name to lower case for the profile storage
+        // REVIEW: This doesn't work for UTF8...
```

Comment:
True. I will make this WIN32 only.

---
In reply to: [125370583](https://github.com/Microsoft/ChakraCore/pull/3284#discussion_r125370583) [](ancestors = 125370583)